### PR TITLE
bring PGEN output dir changes from ah_var_store

### PR DESCRIPTION
**Pull request by rsasch**
Port over changes to `ah_var_store` to make input `output_gcs_dir` behave the same way for PGEN as VCF extract.

Successful run here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/4862a35d-7ac3-421e-9b91-86acaf2b8376

### CODE REVIEW

File: scripts/variantstore/InputValidation.ipynb
Comment (rsasch) [inline scripts/variantstore/InputValidation.ipynb:1]: no idea why this shows up as modified

Comment (mcovarr) [reply scripts/variantstore/InputValidation.ipynb:1]: remove from PR?

File: scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl [hunk 1]
```diff
@@ -130,22 +130,20 @@ workflow GvsExtractCallsetPgen {
...
 
     Int effective_scatter_count = if defined(scatter_count) then select_first([scatter_count])
-                                  else if GetNumSamplesLoaded.num_samples < 100 then 100 # Quickstart
-                                       else if GetNumSamplesLoaded.num_samples < 1000 then 500
-                                            else if GetNumSamplesLoaded.num_samples < 5000 then 1000
-                                                 else if GetNumSamplesLoaded.num_samples < 20000 then 2000 # Stroke Anderson
-                                                      else if GetNumSamplesLoaded.num_samples < 50000 then 10000
-                                                           else if GetNumSamplesLoaded.num_samples < 100000 then 20000 # Charlie
-                                                                 else 34000
+                                  else if GetNumSamplesLoaded.num_samples < 5000 then 1
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl:133]: is this intentional?

Comment (rsasch) [reply scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl:133]: see https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl#L132

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl:133]: PGEN extract doesn't work exactly like VCF extract, there's a separate post stage to merge PGEN shards by chromosome. Has this actually been run at < 5000 scale? I'm not sure the way the PGEN extract shards are configured now that they would survive scattering so narrowly.

Comment (rsasch) [reply scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl:133]: The test run linked to at the top of this PR is quickstart data (10 samples).

**Review by mcovarr — APPROVED** [review]

**Comment by rsasch** [comment]
converted to a draft to make sure I don't merge before it's safe (due to call caching)

**Pull request merged**